src/main.cpp

```diff
@@ -59,23 +59,19 @@ GLdouble lastFrame = 0.0f;
 
 // Scene
 Scene *scene;
-vector<Particle> particles;
-vector<Constraint *> constraints;
+Physics physicSystem;
 
 // Shaders and rendering 
 ParticleRenderer *particleRenderer;
+bool renderSurfaces = false;
```

Comment:
This feature should probably not have been used as base for this branch. Not that it matters too much, we just need to merge surfacerendering first or live with som unfinished changes in dev.